sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -12,6 +12,7 @@
 
   .control-style {
     background-color: var(--bloom-color-white);
+    background-color: rgba(255, 255, 255, 0.85);
```

Comment:
Gives the zoom controls a slight opacity so you can see there is something underneath

<img width="183" alt="Screenshot 2024-11-12 at 9 58 48 PM" src="https://github.com/user-attachments/assets/94e3b9fa-b23d-43fa-8ad7-a39e9d194e76">
<img width="200" alt="Screenshot 2024-11-12 at 9 58 58 PM" src="https://github.com/user-attachments/assets/c5dee511-7e45-44ac-9132-ee8243348fe3">